sdk/search/search-documents/src/indexModels.ts

```diff
@@ -410,9 +457,12 @@ export interface SearchRequestOptions<
    * This parameter is only valid if the query type is 'semantic'. If set, the query returns answers
    * extracted from key passages in the highest ranked documents. The number of answers returned can
    * be configured by appending the pipe character '|' followed by the 'count-\<number of answers\>' option
-   * after the answers parameter value, such as 'extractive|count-3'. Default count is 1.
+   * after the answers parameter value, such as 'extractive|count-3'. Default count is 1. The
+   * confidence threshold can be configured by appending the pipe character '|' followed by the
+   * 'threshold-\<confidence threshold\>' option after the answers parameter value, such as
+   * 'extractive|threshold-0.9'. Default threshold is 0.7.
```

Comment:
`count` and `threshold` are exposed as top-level options in .NET surface, but we've been exposing the raw string at the moment. I'll update the comment since users are expected to use `AnswersType` now.